cdk/index.ts

```diff
@@ -12,7 +12,7 @@ if (process.env.CDK_STACK_PROPS_FILE_PATH === undefined) {
 }
 
 const stackPropsData = JSON.parse(
-  readFileSync(process.env.CDK_STACK_PROPS_FILE_PATH, 'utf8')
+  readFileSync(process.env.CDK_STACK_PROPS_FILE_PATH, 'utf8'),
```

Comment:
If this is a JSON file, you can use built-in features for this:

```
const stackPropsData = require(process.env.CDK_STACK_PROPS_FILE_PATH);
```

but it will do the same thing.